test/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting.Tests/EmbeddingTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Extensions.AI.Evaluation.Reporting.Formats.Html;
+using Microsoft.TestUtilities;
+using Xunit;
+
+namespace Microsoft.Extensions.AI.Evaluation.Reporting.Tests;
+
+public class EmbeddingTests
+{
+    [ConditionalFact]
+    public void CIBuildsMustIncludeEmbeddedHTML()
+    {
+        // TF_BUILD should be set in our CI pipeline
+        if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("TF_BUILD")))
+        {
+            throw new SkipTestException("Skipping test because it is not running in CI");
+        }
+
+        Assert.NotEmpty(HtmlReportWriter.HtmlTemplateBefore);
+        Assert.NotEmpty(HtmlReportWriter.HtmlTemplateAfter);
```

Comment:
minor: It may be better to actually test this in a more black-box way by cracking open the assembly and reading the resource string... (Just in case we change how the static fields work in the future but forget to update / remove this test - for example if we switch to using a default / hardcoded template for some reason...)

I don't feel too strongly about this - but if you do end up changing this now, please flip the fields back to private.

Reply:
I'm going to leave it for now, but I think we can continue to iterate.  One other thought I had was to add a command line option that would check the embedded data when running `aieval version`  for example. Then we could run the console command from the pipeline to see if it succeeds. The only problem with this is that console packages multiple TFMs and I don't know how to invoke each TFM separately without installing multiple SDKs.

Reply:
One idea may be to add a test under `Console.Tests` to programmatically invoke the corresponding command in the `aieval` tool. This test would then run for all TFMs for which the tool is built.